generator/source/_includes/icons.html

```diff
@@ -0,0 +1,159 @@
+	<a href="https://www.facebook.com/{{ site.theme_settings.facebook }}" title="{{ site.theme_settings.str_follow_on }} Facebook">
+		<i class="fa fa-fw fa-facebook"></i>
+	</a>
+</li>
+{% endif %}
+
+{% if site.theme_settings.flickr %}
+<li>
+	<a href="https://www.flickr.com/photos/{{ site.theme_settings.flickr }}" title="{{ site.theme_settings.str_follow_on }} Flickr">
+		<i class="fa fa-fw fa-flickr"></i>
+	</a>
+</li>
+{% endif %}
+
+{% if site.theme_settings.github %}
+<li>
+	<a href="https://github.com/{{ site.theme_settings.github }}" title="{{ site.theme_settings.str_follow_on }} GitHub">
+		<i class="fa fa-fw fa-github"></i>
+	</a>
+</li>
+{% endif %}
+
+{% if site.theme_settings.google_plus %}
+<li>
+	<a href="https://plus.google.com/{{ site.theme_settings.google_plus }}" title="{{ site.theme_settings.str_follow_on }} Google+">
+		<i class="fa fa-fw fa-google-plus"></i>
+	</a>
+</li>
+{% endif %}
+
+{% if site.theme_settings.instagram %}
+<li>
+	<a href="http://instagram.com/{{ site.theme_settings.instagram }}" title="{{ site.theme_settings.str_follow_on }} Instagram">
+		<i class="fa fa-fw fa-instagram"></i>
+	</a>
+</li>
+{% endif %}
+
+{% if site.theme_settings.linkedin %}
+<li>
+	<a href="{{ site.theme_settings.linkedin }}" title="{{ site.theme_settings.str_follow_on }} LinkedIn">
+		<i class="fa fa-fw fa-linkedin"></i>
+	</a>
+</li>
+{% endif %}
+
+{% if site.theme_settings.pinterest %}
+<li>
+	<a href="http://www.pinterest.com/{{ site.theme_settings.pinterest }}" title="{{ site.theme_settings.str_follow_on }} Pinterest">
+		<i class="fa fa-fw fa-pinterest"></i>
+	</a>
+</li>
+{% endif %}
+
+{% if site.theme_settings.reddit %}
+<li>
+	<a href="https://www.reddit.com/user/{{ site.theme_settings.reddit }}" title="{{ site.theme_settings.str_follow_on }} Reddit">
+		<i class="fa fa-fw fa-reddit"></i>
+	</a>
+</li>
+{% endif %}
+
+{% if site.theme_settings.soundcloud %}
+<li>
+	<a href="https://soundcloud.com/{{ site.theme_settings.soundcloud }}" title="{{ site.theme_settings.str_follow_on }} SoundCloud">
+		<i class="fa fa-fw fa-soundcloud"></i>
+	</a>
+</li>
+{% endif %}
+
+{% if site.theme_settings.stack_exchange %}
+<li>
+	<a href="{{ site.theme_settings.stack_exchange }}" title="{{ site.theme_settings.str_follow_on }} Stack Exchange">
+		<i class="fa fa-fw fa-stack-exchange"></i>
+	</a>
+</li>
+{% endif %}
+
+{% if site.theme_settings.steam %}
+<li>
+	<a href="http://steamcommunity.com/id/{{ site.theme_settings.steam }}" title="{{ site.theme_settings.str_follow_on }} Steam">
+		<i class="fa fa-fw fa-steam"></i>
+	</a>
+</li>
+{% endif %}
+
+{% if site.theme_settings.tumblr %}
+<li>
+	<a href="http://{{ site.theme_settings.tumblr }}.tumblr.com/" title="{{ site.theme_settings.str_follow_on }} Tumblr">
+		<i class="fa fa-fw fa-tumblr"></i>
+	</a>
+</li>
+{% endif %}
+
+{% if site.theme_settings.twitter %}
+<li>
+	<a href="https://twitter.com/{{ site.theme_settings.twitter }}" title="{{ site.theme_settings.str_follow_on }} Twitter">
```

Comment:
We could perhaps find a way to search for sawtooth tweets instead of just Hyperledger.

Reply:
Research project. Maybe possible.